manifests/cf-manifest/manifest/000-base-cf-deployment.yml

```diff
@@ -33,7 +33,7 @@ releases:
 stemcells:
   - alias: default
     name: bosh-aws-xen-hvm-ubuntu-trusty-go_agent
-    version: 3232.11
+    version: 3262.2
```

Comment:
The release notes for [239](https://github.com/cloudfoundry/cf-release/releases/tag/v239) have a Recommended BOSH Stemcell Versions section that reads:

- AWS: light-bosh-stemcell-3262.2-aws-xen-hvm-ubuntu-trusty-go_agent
- vSphere: bosh-stemcell-3262.2-vsphere-esxi-ubuntu-trusty-go_agent
- OpenStack: N/A
- BOSH-Lite: bosh-stemcell-3262.2-warden-boshlite-ubuntu-trusty-go_agent
These are soft recommendations; several different versions of the stemcells are likely to work fine with this version of cf-release and the corresponding versions of the additional releases listed below.

We figured using the newest version of the recommendations was only a good thing.